save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityGeneralInfo.kt

```diff
@@ -16,14 +22,34 @@ import react.dom.html.ReactHTML.textarea
 import react.router.dom.Link
 import web.cssom.ClassName
 import web.cssom.rem
+import web.html.ButtonType
 
 import kotlinx.datetime.TimeZone
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
 
 /**
  * [FC] that is used to display some general vulnerability information
  */
 @Suppress("EMPTY_BLOCK_STRUCTURE_ERROR", "MAGIC_NUMBER")
 val vulnerabilityGeneralInfo: FC<VulnerabilityGeneralInfo> = FC { props ->
+
+    val (isEditDisabled, setIsEditDisabled) = useState(true)
```

Comment:
I would do the same as we did for settings: create a data class where you will change several fields, not only this one. And will have one single flag for it.

Right now for each and every change: name, short description, long description, you will create a state.
It's complicated and strange, let's make at least this thing properly

Reply:
Moved the Change button to the very top. The entire vulnerability is now transferred to the state. This is much easier than a separate date class.